akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala

```diff
@@ -6,23 +6,16 @@ package akka.projection.cassandra.internal
 
 import java.util.concurrent.atomic.AtomicBoolean
 
-import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
-import scala.concurrent.Promise
-import scala.concurrent.duration.FiniteDuration
-
-import akka.Done
-import akka.NotUsed
-import akka.actor.ActorSystem
-import akka.actor.ClassicActorSystemProvider
+import akka.{ Done, NotUsed }
+import akka.actor.{ ActorSystem, ClassicActorSystemProvider }
 import akka.annotation.InternalApi
-import akka.projection.Projection
-import akka.projection.ProjectionId
+import akka.projection.{ Projection, ProjectionId }
 import akka.stream.KillSwitches
-import akka.stream.alpakka.cassandra.scaladsl.CassandraSessionRegistry
-import akka.stream.scaladsl.Flow
-import akka.stream.scaladsl.Sink
-import akka.stream.scaladsl.Source
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
+import akka.stream.scaladsl.{ Flow, Sink, Source }
+
+import scala.concurrent.{ ExecutionContext, Future, Promise }
+import scala.concurrent.duration.FiniteDuration
```

Comment:
To reduce merge conflicts when we are so many working on the same small code base we must use the same "optimize imports settings". What we use in akka/akka is single line imports without grouping. That cause least merge conflicts.
cc @renatocaval

Reply:
@patriknw, you mean in IntelliJ? 

I was looking for a rewrite rule in scalafmt, but didn't find any.

Reply:
Created this PR https://github.com/akka/akka-projection/pull/76

Is that what we want, if so. Please, merge and rebase this one with master to reduce conflicts.

Reply:
I've applied the same settings as @renatocaval.  I'll rebase when #76 is merged.

Reply:
I've since reverted the files that mangled imports.